picture-ts/src/lib/ui.ts

```
@@ -0,0 +1,461 @@
+        // Create the appropriate progress display
+        switch (this.style) {
+            case 'spinner':
+                this.spinner = ora({
+                    text: `${this.title}...`,
+                    spinner: {
+                        frames: SPINNER_CHARS
+                    }
+                }).start();
+                break;
+            case 'bar':
+                let barFormat = `${this.title} |${chalk.cyan('{bar}')}| {percentage}% | {value}/{total}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)